lib/tasks/load_data.rake

```diff
@@ -0,0 +1,36 @@
+require 'csv'
+namespace :load_data do
+  task :csvs => :environment do
+    puts "loading merchants"
+    CSV.foreach('db/data/merchants.csv', :headers => true, :encoding => 'ISO-8859-1:UTF-8') do |row|
+      Merchant.create([
+                      :name => row['name']
+                      ])
+    end
+    puts "loaded #{Merchant.all.count} merchants"
+
+    puts "loading transactions"
+    CSV.foreach('db/data/transactions.csv', :headers => true, :encoding => 'ISO-8859-1:UTF-8') do |row|
```

Comment:
is there any reason to create separate rake tasks for loading each csv? could you see any benefit to separating these out rather than having one long rake task?